examples/with-firebase-hosting/firebaseFunctions.js

```diff
@@ -2,11 +2,10 @@ const { join } = require('path')
 const { https } = require('firebase-functions')
 const { default: next } = require('next')
 
-const isDev = process.env.NODE_ENV !== 'production'
 const nextjsDistDir = join('src', require('./src/next.config.js').distDir)
 
 const nextjsServer = next({
-  dev: isDev,
+  dev: false,
```

Comment:
I've changed that because the development mode caused a lot of confusion previously (see #8893). You cannot run next in development mode inside the cloud function.

Reply:
Great! I cannot recount how many times I have field this question. The app running locally (without being on Cloud Function) can still connect and interact with the Firebase local emulator.